.github/workflows/Packages - EdFi.Ods.Extensions.SK.yml

```diff
@@ -6,8 +6,6 @@
 name: Packages - EdFi.Ods.Extensions.Sk
 
 on:
-  pull_request:
```

Comment:
I removed here because we are not running pull request for any other templates also